ansible_collections/arista/avd/molecule/eos_designs_unit_tests/converge.yml

```diff
@@ -9,20 +9,3 @@
       delegate_to: 127.0.0.1
       ansible.builtin.import_role:
         name: arista.avd.eos_designs
-
```

Comment:
line 3 I also replaced `hosts: all` by `hosts: EOS_DESIGNS_UNIT_TESTS` when I introduced the negative tests - we may want to revert this back for consistency

Reply:
good suggestion, I have updated both to use `hosts: all`